src/editors/utils/index.ts

```diff
@@ -4,4 +4,5 @@ export { default as camelizeKeys } from './camelizeKeys';
 export { default as removeItemOnce } from './removeOnce';
 export { default as formatDuration } from './formatDuration';
 export { default as snakeCaseKeys } from './snakeCaseKeys';
+export { default as convertMarkdownToXml } from './convertMarkdownToXML';
```

Comment:
We no longer require the use of default exports, so you can make this a regular export in `convertMarkdownToXML.js` and then simplify this to `export { convertMarkdownToXml } from './convertMarkdownToXML';` if you want. That's more flexible and consistent. But this is optional.